bemani/backend/iidx/base.py

```diff
@@ -364,8 +365,9 @@ def update_score(
             ex_score = max(ex_score, oldscore.points)
             scoredata = oldscore.data
             scoredata.replace_int('clear_status', max(scoredata.get_int('clear_status'), clear_status))
+            if miss_count != -1:
+                scoredata.replace_int('miss_count', min(scoredata.get_int('miss_count'), miss_count))
```

Comment:
It looks like this will still fail if a user already has a -1 in their miss count from a previous bug.

Reply:
Right thanks. Let me think up a solution

Reply:
Should be fixed now